### PR TITLE
Adds support for cassandra integration tests

### PR DESCRIPTION
**Pull request by vinamartin**
Refactors Junit 5 tests back to Junit 4, adds Cassandra itests. 
Explicitly defines dependency versions. 

@Kjames5269 @jhunzik

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,11 +36,10 @@ subprojects {
 	tasks.withType(Test) {
 		testlogger {
 			theme = 'mocha'
-			showStandardStreams = false
-			showSkippedStandardStreams = false
+			showStandardStreams = true
```
Comment (vinamartin) [inline build.gradle]: ✏️ I might turn these back to false. 
It turns out the logging isn't as great as I hoped.

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by jhunzik** [comment]
LGTM, I'm going to hold off on the review until the actual itest gets in.

**Comment by vinamartin** [comment]
build now

**Comment by vinamartin** [comment]
build now

File: build.gradle [hunk 2]
```diff
@@ -155,17 +152,13 @@ subprojects {
 		}
 
 		compileOnly "org.projectlombok:lombok:1.18.6"
-		annotationProcessor "org.projectlombok:lombok"
-		testImplementation "org.springframework.boot:spring-boot-starter-test"
```
Comment (Kjames5269) [inline build.gradle]: `2.1.5.RELEASE` ?

**Pull request merged**